app/assets/javascripts/discourse/app/components/modal/login.hbs

```diff
@@ -58,7 +58,7 @@
               />
             </WelcomeHeader>
           {{/if}}
-          <Modal::Login::LocalLoginForm
+          <LocalLoginForm
```

Comment:
Do any plugins or theme components extend the old modal login form? Would be good to check.

Reply:
I didn't find any